### PR TITLE
add yehuda kahan

### PR DESCRIPTION
**Pull request by yehuda-kahan**

### CODE REVIEW

File: people/yehdua-kahan.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "יהודה קאהן",
+    "github": "yehuda-kahan",
+    "facebook": "",
+    "twitter": "",
+    "linkedin": "https://www.linkedin.com/in/yehuda-kahan-00294418a/",
+    "stackoverflow": "",
+    "website": "",
```
Comment (MichalPorag) [inline people/yehdua-kahan.json:8]: You can remove this lines

File: people/yehdua-kahan.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "יהודה קאהן",
+    "github": "yehuda-kahan",
+    "facebook": "",
+    "twitter": "",
```
Comment (MichalPorag) [inline people/yehdua-kahan.json:5]: You can remove this lines

**Review by MichalPorag — CHANGES_REQUESTED** [review]
Thanks for your contribution ❤️

Please remove the marked lines.

File: people/yehdua-kahan.json (hunk 2 above)
Comment (yehuda-kahan) [reply people/yehdua-kahan.json]: Thanks, I did it

**Pull request merged**